pkg/publish.go

```diff
@@ -97,3 +124,104 @@ func Publish(file string, rootID string, dryRun bool) (id string, err error) {
 	}
 	return c.ID, nil
 }
+
+func GetImageSrcsFromHTML(content string) []string {
+	var imageSrcs []string
+	z := html.NewTokenizer(strings.NewReader(content))
+	for {
+		tt := z.Next()
+		t := z.Token()
+		switch tt {
+		case html.ErrorToken:
+			return imageSrcs
```

Comment:
If t.Err() is not `io.EOF`, we'll want to return that `error` here (malformed HTML, etc.).